catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Service.js

```diff
@@ -333,5 +333,46 @@ define(function (require) {
         }]
     });
 
+    Service.Fanout = Backbone.Model.extend({
+        url: "/admin/jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui,version=2.3.0/getProperties/ddf.catalog.impl.operations.SourceOperations",
+        saveUrl: "/admin/jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui,version=2.3.0/update",
+        defaults: function () {
+            return {
+                fanoutSourceList:  [],
+                fanoutTagWhitelist:  ['resource'],
+                invertFanoutList: false
+            };
+        },
+        parse: function(raw){
+            if(raw) {
+                return raw.value;
+            }
+            return raw;
+        },
+        save: function () {
+            var deferred = $.Deferred();
```

Comment:
What is this exactly being used for? Can we just use the object returned from the $.ajax request since it implements the promise interface already?